docs/UserGuide.md

```diff
@@ -96,7 +96,7 @@ For advanced users, the [Command summary](#command-summary) section gives you an
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME e/EMAIL`, `e/EMAIL n/NAME` is also acceptable.
 
-* If a command requires `INDEX` as an input, only one input for `INDEX` is expected. `INDEX` **must** be an integer from 1 to 10000. <br>
+* If a command requires `INDEX` as an input, only one input for `INDEX` is expected. `INDEX` **must** be an integer from 1 to 10,000. <br>
```

Comment:
not sure if i remembered correctly the index is from 1 to 9999 (less than 10000)

Reply:
Max is 10000 I believe. I wrote the test cases with it too. Can download our jar file to confirm again